packages/autocomplete-tools/@types/fig/index.d.ts

```diff
@@ -3,25 +3,106 @@ export {};
 
 declare global {
   export namespace Fig {
-    // All the available templates
     type TemplateStrings = "filepaths" | "folders" | "history";
 
+    /**
+     * A template which is a single TemplateString or an array of TemplateStrings.
```

Comment:
Based on TSDoc in this file, I think the trailing dot should be removed
```suggestion
     * A template which is a single TemplateString or an array of TemplateStrings
```